multicluster/cmd/gateways.go

```diff
@@ -117,6 +117,12 @@ func newGatewaysCommand() *cobra.Command {
 					continue
 				}
 
+				// If the Link resource does not specify a gateway, do not
+				// process metrics
+				if parsedMetrics["gateway_enabled"] == nil {
```

Comment:
the comments suggest that a value of "0" means disabled so maybe this check should be `!= "1"`?